composer.json

```diff
@@ -45,6 +45,8 @@
         "jms/di-extra-bundle": "~1.7",
         "sensio/generator-bundle": "~2.3|~3.0",
         "symfony/yaml": "~2.3|~3.0",
+        "symfony/security-acl": "~2.4|~3.0",
+        "phpunit/phpunit": "~4.3",
```

Comment:
> If someboday checks out SonataAdmin, this person should be able to run tests. For this, he need of course phpunit. But which version? Exactly the one that is used in composer.json of course!
>
> That has nothing to do with "globally installed phpunit" or what ever. It doesnt bother me, where you got installed your phpunit... Put its a fact, that if I use another phpunit version than you, I cannot run correct tests...

100 % agree with you on this. I raised this problem myself before and was told it was to save time on the build